Dockerfile.perses

```diff
@@ -0,0 +1,33 @@
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_golang_1.23 AS build-env
+USER root
+RUN dnf install -y mailcap && dnf clean all && rm -rf /var/cache/dnf
+
+WORKDIR /app
+
+COPY perses/ .
+
+RUN go mod tidy && go mod vendor 
+RUN make build-api 
+RUN make build-cli
+
+FROM registry.redhat.io/ubi8/ubi-minimal:8.10-1130
+
+LABEL maintainer="Observability UI team <team-observability-ui@redhat.com>"
+
+USER nobody
+
+COPY --from=build-env --chown=nobody:nobody /app/bin/perses                         /bin/perses
+COPY --from=build-env --chown=nobody:nobody /app/bin/percli                         /bin/percli
+COPY --from=build-env --chown=nobody:nobody /app/LICENSE                            /bin/LICENSE
+COPY --from=build-env --chown=nobody:nobody /app/plugins-archive/                   /etc/perses/plugins-archive/
```

Comment:
This is not needed on this version of Perses and it might fail as it does not exist